### ISSUE TITLE
Segmente mit parking=separate, aber weiteren parking-Tags werden als Parkstreifen prozessiert

### ISSUE THREAD

**Issue by SupaplexOSM** [osmberlin/osm-parking-processing#77]
Ein Mapper aus Hannover hat mich auf diese Stelle ([OSM](https://www.openstreetmap.org/way/1157003706) | [Atlas](https://radverkehrsatlas.de/regionen/parkraum?lat=52.39816&lng=9.82655&zoom=19&theme=parking&bg=default&config=!(i~parking~topics~!(i~parking~a~~s~!(i~default~a)(i~presence~a~_F)(i~raw~a~_F))(i~parkingPoints~a~_F~s~!(i~default~a))(i~parkingAreas~a~~s~!(i~default~a)(i~position-separate~a~_F))(i~parkingDebug~a~~s~!(i~default~a))(i~parkingStats~a~_F~s~!(i~default~a~~f~!(i~admin*_level~o~!(i~*4~a~_F)(i~*9~a)(i~*10~a~_F)))(i~length~a~_F~f~!(i~admin*_level~o~!(i~*4~a~_F)(i~*9~a)(i~*10~a~_F))))(i~landuse~a~_F~s~!(i~default~a)))~)) aufmerksam gemacht, an der `parking:right=separate` + `parking:right:orientation=perpendicular` getaggt ist.

Eigentlich sollten dann dort nur die separaten Flächen erscheinen (deren Prozessierung gerade noch überarbeitet wird), stattdessen wird jedoch die ganze Linie als Querparken dargestellt. Also: wenn das Primärtag `parking=separate` ist, dann sollten alle anderen Attribute an der Linie ignoriert werden (auch wenn sie dort eigentlich nicht erscheinen sollten).

**Comment by joshinils** [comment]
ich würde sagen das ist auch ein fall für ein fehler-layer.
das ist doch ein tagging fehler, und sollte als solcher behoben werden, oder?

**Comment by SupaplexOSM** [comment]
Ja, würde ich auch sagen, aber unabhängig davon sollten solche Fehler nicht prozessiert werden ;)

**Comment by SupaplexOSM** [comment]
P.S. Wobei ich es nicht als "Fehler" bezeichnen würde, denn es ist nicht "falsch", nur ungünstig getaggt.
